### PR TITLE
Migrating pkg/kubelet/winstats to structured logging

### PR DESCRIPTION
**Pull request by hexxdump**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Migrate pkg/kubelet/winstats to structured logging

#### Which issue(s) this PR fixes:
Part of #98976

#### Special notes for your reviewer:
NONE

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

`https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
`

/sig node instrumentation
/priority important-soon

### CODE REVIEW

**Review by navidshaikh — COMMENTED** [review]
lgtm
/cc @ehashman @serathius

**Review by ehashman — COMMENTED** [review]
/ok-to-test
/triage accepted
/lgtm

Can you fix your release note? The field needs to start with ```release-note rather than using single backticks.

File: pkg/kubelet/winstats/perfcounter_nodestats.go [hunk 1]
```diff
@@ -168,25 +168,25 @@ func (p *perfCounterNodeStatsClient) collectMetricsData(cpuCounter, memWorkingSe
...
 	if err != nil {
-		klog.Errorf("Unable to get cpu perf counter data; err: %v", err)
+		klog.ErrorS(err, "Unable to get cpu perf counter data")
 		return
 	}
 
 	memWorkingSetValue, err := memWorkingSetCounter.getData()
 	if err != nil {
-		klog.Errorf("Unable to get memWorkingSet perf counter data; err: %v", err)
+		klog.ErrorS(err,"Unable to get memWorkingSet perf counter data")
```
Comment (ehashman) [inline pkg/kubelet/winstats/perfcounter_nodestats.go:177]:
```suggestion
		klog.ErrorS(err, "Unable to get memWorkingSet perf counter data")
```

gofmt is failing on this line.

Comment (hexxdump) [reply pkg/kubelet/winstats/perfcounter_nodestats.go]: Thanks @ehashman. Changed as suggested.

**Comment by navidshaikh** [comment]
@hexxdump : can you squash the commits?

**Review by ehashman — COMMENTED** [review]
/lgtm

**Pull request merged**